<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WithdrawItemDialog</class>
 <widget class="QDialog" name="WithdrawItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Withdraw Item</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="itemNameInput">
     <property name="placeholderText">
      <string>Item Name</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="quantityInput">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="observationInput">
     <property name="placeholderText">
      <string>Observations</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateTimeEdit" name="dateTimeInput">
     <property name="displayFormat">
      <string>dd/MM/yyyy HH:mm:ss</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="withdrawButton">
     <property name="text">
      <string>Retirar do estoque</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
